packages/kbn-i18n/src/server/__tests__/i18n.js

```diff
@@ -0,0 +1,213 @@
+/*
```

Comment:
__issue:__ while it's not too late please switch to Jest tests instead :) Take a look at how we use them in other packages (e.g. no need in `__tests__` folder).

Sorry, I should have made that clearer before - we're moving away from Mocha and try to use Jest for all new tests we write.

Reply:
Ok, it's not a problem, I just moved the old tests that were written by previous developers to the new package. I will rewrite them using Jest.